src/main/java/org/bladerunnerjs/contrib/contentplugin/appcache/AppcacheManifestBuilder.java

```diff
@@ -52,16 +45,11 @@ public AppcacheManifestBuilder(BRJS brjs, BundleSet bundleSet, String brjsVersio
 	 */
 	public String getManifest(RequestMode requestMode) throws PropertiesException, ContentProcessingException, ConfigException, MalformedTokenException
 	{
-		String manifest = getManifestHeader();
-        if(version != null)
```

Comment:
Well I didn't mean leave it *exactly* as it was, it should be `if(!version.equals("dev"))` or something similar. The logic is that if it's dev mode with appcache disabled (i.e. we didn't specify a version so it's using the default), then we don't want to write all the files to the manifest.
The reason being is that appcache will try to cache all of those files again before it loads the new, appcache-disabled page, and that's slow.